testsuite/tests/singlecluster/authorino/caching/metadata/test_ttl.py

```diff
@@ -9,30 +9,28 @@
 
 pytestmark = [pytest.mark.authorino]
 
-
-@pytest.fixture(scope="module")
-def cache_ttl():
-    """Returns TTL in seconds for Cached Metadata"""
-    return 5
+CACHE_TTL = 5
```

Comment:
Just to educate myself, what is the reason for using const instead of fixture here? I assume that fixture processing requires a bit more CPU/RAM and const works equally well. Was there any other reason?

Reply:
I thinking in general in our testsuite we use global constants if they are used just in one test file and constant in fixture if it would be used in multiple files. At least thats how I try to write code.